docs/index.asciidoc

```diff
@@ -31,6 +31,8 @@ Requirements:
 
 S3 outputs create temporary files into the OS' temporary directory, you can specify where to save them using the `temporary_directory` option.
 
+IMPORTANT: For configurations containing multiple s3 outputs with the restore option enabled, each output should define its own 'temporary_directory'
+
```

Comment:
Here's how this renders:
<img width="894" alt="screen shot 2019-02-25 at 11 53 17 am" src="https://user-images.githubusercontent.com/35154725/53354057-08a94f00-38f4-11e9-941b-a02208c4a6db.png">